raiden/tests/utils/genesis.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8
+from raiden.settings import GAS_LIMIT_HEX
+GENESIS_STUB = {
+    'config': {
+        'homesteadBlock': 0,
+        'eip150Block': 0,
+        'eip150Hash': '0x0000000000000000000000000000000000000000000000000000000000000000',
+        'eip155Block': 0,
+        'eip158Block': 0,
+    },
+    'nonce': '0x0',
+    'mixhash': '0x0000000000000000000000000000000000000000000000000000000000000000',
+    'difficulty': '0x1',
+    'coinbase': '0x0000000000000000000000000000000000000000',
+    'timestamp': '0x00',
+    'parentHash': '0x0000000000000000000000000000000000000000000000000000000000000000',
+    'extraData': '0x' + 'raiden'.encode('hex'),
+    'gasLimit': GAS_LIMIT_HEX,
+    # add precompile addresses with minimal balance to avoid deletion
```

Comment:
what is a precompile? Precompile is a verb which does not describe anything. Precompiled is an adjective. The meaning is still not clear.

Are they precompiled contracts? Are they just addresses with preallocated balances? If so just call them preallocated.